mindsdb/integrations/handlers/tpot_handler/Manual_QA.md

```diff
@@ -0,0 +1,37 @@
+# Welcome to the MindsDB Manual QA Testing for TPOT Handler
```

Comment:
In addition to this doc, can you please fill out [this form template](https://docs.google.com/document/d/1yOpRkfjhtL6KmueC0dIvn6rZ-PD3b9XdIa7Ywnyp6Jk/edit?usp=sharing) and add it as a `README.md`?